smartgymapi/models/sport_schedule.py

```diff
@@ -13,16 +15,26 @@ class SportSchedule(Base, LineageBase):
     user_id = Column(UUIDType, ForeignKey('user.id'))
     name = Column(String(100))
     reminder_minutes = Column(Integer)
-
+    time = Column(Time)
+    weekdays = Column(Integer)
+    is_active = Column(Boolean)
 
     def set_fields(self, data=None):
         for key, value in data.items():
             setattr(self, key, value)
 
 
-def list_sport_schedules():
-    return DBSession.query(SportSchedule)
+def list_sport_schedules(user_id=None):
+    q = DBSession.query(SportSchedule)
```

Comment:
lijkt me dat user_id hier verplicht is of is er een situatie waar we alle sport_schedules in de db willen zien?